src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -1203,6 +1203,28 @@
             </div>
           </div>
         </ng-container>
+
+      <!-- RGW QAT Compression -->
+        @if(serviceForm.controls.service_type.value === 'rgw') {
+        <div class="form-item"
+             formGroupName="qat">
+            @for(property of rgwQatList; track property) {
+            @if(property === 'compression') {
```

Comment:
oh okay, i see the `property` now. let's not use this. make it simple for now and define the `compression` there in the ts as I mentioned and then use it as it is.